### DISCUSSION TITLE
pnpm monorepo `pnpm -r`

### DISCUSSION THREAD

**Discussion by aussieboi** [pnpm/pnpm#5763] [Q&A] [answered]
Hello, I have search issues and discussion but haven't found similar example.

i have monorepo setup as : 
https://stackblitz.com/edit/node-nrtf1z?file=apps/test/package.json

if I include package  in app as following: 

```
"@demo/ui": "workspace:*"
```

if I run `pnpm run all:dev (pnpm -r dev) ` pnpm ignores app and doesn't build app like it is filtered out? I want to run all apps and packages in parallel so i don't have to open now terminal and run `pnpm run web:dev (pnpm --filter @demo/app dev)`, for better DX.

am I missing some configuration or pnpm is limited in this case? 

Thanks a lot

**Comment by zkochan** [comment, marked as answer]
Use the [`--parallel`](https://pnpm.io/cli/run#--parallel) option:

```
pnpm --parallel -r dev
```

**Reply by aussieboi** [reply]
Awesome tnx :)
